lale/lib/sklearn/bagging_classifier.py

```diff
@@ -246,6 +317,41 @@
     ],
 }
 
+_input_score_schema = {
+    "description": "Return the mean accuracy on the given test data and labels.",
+    "type": "object",
+    "required": ["X", "y"],
+    "properties": {
+        "X": {
+            "type": "array",
+            "items": {
+                "type": "array",
+                "items": {"type": "number"},
+            },
+            "description": "Test samples.",
+        },
+        "y": {
```

Comment:
The schema for `y` is consistent with what we specified in fit and predict for BaggingClassifier. But it might be too restrictive, because in practice, class labels are often not numbers but strings or booleans. I recommend changing it to an `anyOf`, similar to what we have in many other classifiers such as [DecisionTreeClassifier](https://github.com/IBM/lale/blob/master/lale/lib/sklearn/decision_tree_classifier.py#L214-L221).

Reply:
Makes sense, will do.